crates/bevy_pbr/src/light/mod.rs

```diff
@@ -1825,12 +1826,12 @@ pub fn update_spot_light_frusta(
         // by applying those as a view transform to shadow map rendering of objects
         let view_backward = transform.back();
 
-        let spot_view = spot_light_view_matrix(transform);
-        let spot_projection = spot_light_projection_matrix(spot_light.outer_angle);
-        let view_projection = spot_projection * spot_view.inverse();
+        let spot_view_from_world = spot_light_view_from_world(transform);
+        let spot_clip_from_view = spot_light_clip_from_view(spot_light.outer_angle);
+        let clip_from_world = spot_clip_from_view * spot_view_from_world.inverse();
```

Comment:
This should then be clip from world = clip from view * world from view inverse. World from view inverse meaning the same as view from world.